smg-automotive/components-pkg
# feat: popover tooltip

References [VSST-1956](https://autoricardo.atlassian.net/browse/VSST-1956)

## Motivation and context

The top vehicle management page displays a table of listings. Some of these listings cannot be prioritised as they are either lacking images or had an accident. In case of these listings, we should display a popover tooltip explaining why the listing cannot be prioritised. The tooltip should be interactable in case it contains a link/button (the popover should stay open if a user is hovering over it).

## Before

No interactable tooltip component.

## After

<img width="1342" alt="Screenshot 2024-01-23 at 16 54 28" src="https://github.com/smg-automotive/components-pkg/assets/141041162/0c569e02-2ff8-4fd8-9034-fb0fe3b8fab4">

## How to test

- https://popover-tooltip-components-pkg.branch.autoscout24.dev/?path=/story/components-overlay-popover--popover
- should work both on mobile and desktop devices (stay open on interaction)

[VSST-1956]: https://autoricardo.atlassian.net/browse/VSST-1956?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
